### PR TITLE
Bump the changelog for 0.6.11

### PR DESCRIPTION
**Pull request by jonas-schievink**
Unfortunately I already published it. Should we publish 0.6.12 just for this fix?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -467,6 +469,7 @@ section                size        addr
 Initial release
 
 [Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.10...HEAD
```
Comment (adamgreig) [inline CHANGELOG.md]: This line also needs updating to 0.6.11

**Comment by therealprof** [comment]
I guess we can live with the slight mismatch of CHANGELOG.md and the release.

**Comment by adamgreig** [comment]
> Should we publish 0.6.12 just for this fix?

I don't think we need to, the changelog is presumably very rarely viewed from the released crate.

**Comment by therealprof** [comment]
Would be great to have the tag on the right version though...

**Pull request merged**